inflammation/models.py

```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
```

Comment:
Could be more descriptive eg follow same pattern as other functions: `Calculate the standard deviation of a 2d inflammation data array`

Reply:
It can most certainly be made more descriptive. I shall follow your advice and change the description to be more descriptive